src/recordlinker/linking/mpi_service.py

```diff
@@ -107,3 +107,25 @@ def insert_blocking_keys(
     if commit:
         session.commit()
     return values
+
+def get_all_algorithm_labels(session: orm.Session) -> list[str]:
+    """
+    Gets a list of algorithms from the MPI database.
+    returns: list of all labels column from algorithms table
+    """
+    algorithms_list = session.query(models.Algorithm.label).all()
```

Comment:
[scalars](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.scalars) can be a shortcut here if you want, nothing wrong with this implementation, but just so you are aware.

```python
return session.scalars(session.query(models.Algorithm.label)).all()
```

Reply:
Good callout. I like this way much better. Also, I got a deprecation warning when running tests so I went ahead and switched it to use `select` instead of `session.query`. Mainly just to get rid of the warning

Reply:
Yeah, I've seen a few of those warnings as well in the past.  The 2.0 style is a bit different, but we should probably start moving in that direction https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#migration-orm-usage